flux-typeck/src/fixpoint.rs

```diff
@@ -122,56 +124,63 @@ where
         did: DefId,
         constraint: fixpoint::Constraint<TagIdx>,
     ) -> Result<(), Vec<Tag>> {
-        let kvars = self
-            .fixpoint_kvars
-            .into_iter_enumerated()
-            .map(|(kvid, sorts)| fixpoint::KVar(kvid, sorts))
-            .collect_vec();
+        fixpoint_span!(self.genv.tcx, did).in_scope(|| {
+            tracing::debug!(event = "fixpoint start");
```

Comment:
It seems like `tracing_timing` requires events to be emitted since those are what it collects in its histograms.

Putting "start" and "end" events seems a little inelegant though, so I wonder if I'm doing something wrong. Perhaps we only need the "end" event?